qemu/tests/cfg/netperf_udp_perf.cfg

```diff
@@ -0,0 +1,59 @@
+- netperf_udp_perf: install setup image_copy unattended_install.cdrom
+    #only linux
+        ver_cmd = rpm -qa qemu-kvm-rhev && rpm -qa qemu-kvm
+        mem_fixed = 4096
+    type = netperf_udp_perf
+    kill_vm = yes
+    image_snapshot = yes
+    setup_ksm = no
+    take_regular_screendumps = no
+    store_vm_register = no
+    nics += ' nic2'
+    netdst_nic1 = private
+    netdst_nic2 = switch
+    mac_nic2= 9a:37:37:37:37:6e
```

Comment:
Please specify it privately, here leave it be null.

Reply:
I have udpated.